tools/load_database_dump.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script can be used to load downloaded database dump.
+
+# Import utility functions
+# shellcheck source=./tools/_functions.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
+require_installed
+
+SQL_DATA_PATH=$1
+
+bash "${DEV_TOOL_DIR}/prune_database.sh"
+
+docker run -d --name "${DOCKER_CONTAINER_NAME}" -e "POSTGRES_USER=integreat" -e "POSTGRES_PASSWORD=password" -e "POSTGRES_DB=integreat" -v "${BASE_DIR}/.postgres:/var/lib/postgresql" -p "${INTEGREAT_CMS_DOCKER_LISTEN_PORT}":"${INTEGREAT_CMS_DB_PORT}" postgres > /dev/null
+wait_for_docker_container
+
+docker exec -i "${DOCKER_CONTAINER_NAME}" psql -U integreat < "${SQL_DATA_PATH}"
```

Comment:
Please do not assume docker 😅

Reply:
Edit: I think it might be easiest to adapt this line from `loadtestdata.sh`? 
```bash
deescalate_privileges integreat-cms-cli loaddata "${PACKAGE_DIR}/cms/fixtures/test_data.json" --verbosity "${SCRIPT_VERBOSITY}"
```

Reply:
`loaddata` probably won't work since it only can load json exports that were created with `dumpdata`.
There is however the management command [dbshell](https://docs.djangoproject.com/en/5.0/ref/django-admin/#dbshell) which could reduce the whole dev tool to:
```
./tools/integreat-cms-cli dbshell path_to_db_dump.sql
```
and migrate & create_superuser can be run subsequently via the shell, too.
In my opinion, we don't necessarily need an extra dev tool for this.